src/engine.rs

```diff
@@ -44,7 +44,7 @@ impl Engine for PowEngine {
     let rt = PowEngine::build_rt();
 
     {
-      let time_til_publishing = Duration::from_secs(node.config.seconds_between_blocks);
+      let time_til_publishing = Duration::from_millis(500);
```

Comment:
This is a change in behavior which sounds like it needs to be a change in configuration? I need more background here to understand what's going on.

Reply:
FTR the config value isn't necessary inside this function anymore. Changing actual to expected difficulty goes hand-in-hand with the hard-coded interval above.